README.md

```diff
@@ -1,21 +1,85 @@
-### thanks for 100 stars :)
-
 ```
  _   _____   __   ____  ___  ___   _  ________                
 | | / / _ | / /  / __ \/ _ \/ _ | / |/ /_  __/__________  ____
 | |/ / __ |/ /__/ /_/ / , _/ __ |/    / / / /___/ __/ _ \/ __/
 |___/_/ |_/____/\____/_/|_/_/ |_/_/|_/ /_/     /_/ / .__/\__/ 
                                                   /_/         
 ```
-[![Discord](https://img.shields.io/badge/discord-join-7389D8?style=flat&logo=discord)](https://discord.gg/uGuswsZwAT)
-<span class="badge-buymeacoffee">
-<a href="https://ko-fi.com/colinh" title="Donate to this project using Buy Me A Coffee"><img src="https://img.shields.io/badge/buy%20me%20a%20coffee-donate-yellow.svg" alt="Buy Me A Coffee donate button" /></a>
-</span>
+[![Discord][discord-shield]][discord-url]
+[![Stars][stars-shield]][stars-url]
+[![Releases][releases-shield]][releases-url]
+[![Kofi][kofi-shield]][kofi-url]
+[![Language][language-shield]][language-url]
+[![License][license-shield]][license-url]
+
+  <ol>  
+    <li><a href="#about">About</li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#support">Support</a></li>
+    <li><a href="#disclaimer">Disclaimer</a></li>
+  </ol>
+ 
+ 
+## About
+
+
+ Valorant RPC (Rich Presence) allows you to show in-game details such as the current score in your Discord Profile! Additional features include showing current map, agent, idle status, etc.
+
+ 
+ <a>
+    <img src="assets/Demo1.png" alt="Demo" width="205" height="112">
+    <img src="assets/Demo2.png" alt="Demo" width="205" height="112">
 
-# Installation
-- Download the latest [release](https://github.com/colinhartigan/valorant-rpc/releases/) and run it.
+ </a>

```

Comment:
no need for <a> tag, if you want to redirect to url, you need to <a> tag every image, not whole stuff